### PR TITLE
Removing workbook.functions from tutorial

### PR DESCRIPTION
**Pull request by AlexJerabek**
This API is no longer "maker-safe".

### CODE REVIEW

File: docs/tutorials/excel-office-scripts-tutorial.md [hunk 1]
```diff
@@ -190,27 +190,20 @@ Over the rest of the tutorial, we will normalize this data using a script. First
 
     ![The console log displaying the the output "-20.05", nested under two arrays.](../images/tutorial-4.png)
 
-## Apply the absolute value function to a cell
+## Adjust the value of a cell
```
Comment (kbrandl) [inline docs/tutorials/excel-office-scripts-tutorial.md]: Nit: consider changing "Adjust" -> "Modify"

File: docs/tutorials/excel-office-scripts-tutorial.md [hunk 2]
```diff
@@ -190,27 +190,20 @@ Over the rest of the tutorial, we will normalize this data using a script. First
...
+      // Run the `Math.abs` function with the value at D2 and apply that value back to D2.
+      range.values = [[Math.abs(range.values[0][0])]];
     ```
 
 2. The value of cell **D2** should now be positive.
 
-You may have noticed we're now calling `context.sync()` twice. We need to read the values in the cell first. Then, we need to read the result of the absolute value function. We could skip that second `load` and `sync` if we calculated the absolute value locally.
-
-## Apply the absolute value function to a column
+## Adjust the values of a column
```
Comment (kbrandl) [inline docs/tutorials/excel-office-scripts-tutorial.md]: Nit: consider changing -> "Modify values in a column"

File: docs/tutorials/excel-office-scripts-tutorial.md [hunk 3]
```diff
@@ -190,27 +190,20 @@ Over the rest of the tutorial, we will normalize this data using a script. First
 
     ![The console log displaying the the output "-20.05", nested under two arrays.](../images/tutorial-4.png)
 
-## Apply the absolute value function to a cell
+## Adjust the value of a cell
 
-Now that we can read data, let's use it to modify the workbook. We'll make the value of the cell **D2** positive by using Excel's built-in absolute value function.
+Now that we can read data, let's use it to modify the workbook. We'll make the value of the cell **D2** positive with the `Math.abs` function. The [Math](https://developer.mozilla.org/docs/web/javascript/reference/global_objects/math) object contains many functions to which your scripts have access.
```
Comment (kbrandl) [inline docs/tutorials/excel-office-scripts-tutorial.md]: Nit: in the first sentence, consider replacing "use it" with "use the script"

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6f1f2d9](https://github.com/OfficeDev/office-scripts-docs/commits/6f1f2d99f3bba7cc1817309377feb4341d04e4b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Concepts/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-office-scripts-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Concepts/docs/tutorials/excel-office-scripts-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C11%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912111954096186-34%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7bHplmNRiJHlwQ%2FMzJLJMYG48VdFcwOu8limDocscQg%3D&st=2019-12-11T19%3A50%3A24Z&se=2020-01-11T19%3A55%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**